TerrariaServerAPI/TerrariaApi.Server/ServerApi.cs

```diff
@@ -311,6 +316,9 @@ internal static void LoadPlugins()
 					if (!InvalidateAssembly(assembly, fileInfo.Name))
 						continue;
 
+					if (MaliciousAssembly(assembly))
```

Comment:
Should there be an "-unsafe" command line flag a server owner can supply that overrides this behavior?

Reply:
Given the number of people who bypass the checks on versions, I'd say that it's unnecessary atm.